AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerService.kt

```diff
@@ -345,4 +353,22 @@ interface TestRunnerService {
             const val TEXTPROTO = "textproto"
         }
     }
+
+    /**
+     * Represents the issue detected while running tests in a testMatrix in FTL
+     */
+    data class TestIssue(
+        /**
+         * Error message describing the issue
+         */
+        val errorMessage: String,
+        /**
+         * Severity of issue
+         */
+        val severity: String?,
```

Comment:
Is there a reason we are using strings and not types that are generated in dev.androidx.ci.generated.ftl.* ?

Reply:
Yeah, those classes are internal and can't be used here directly. That's why we always create data classes here to only expose the relevant fields and not the entire output.